src/Api/Controllers/CollectionsController.cs

```diff
@@ -109,7 +109,7 @@ public async Task<ListResponseModel<CollectionResponseModel>> Get(Guid orgId)
         var readAllAuthorized = (await _authorizationService.AuthorizeAsync(User, CollectionOperations.ReadAll(orgId))).Succeeded;
         if (readAllAuthorized)
         {
-            orgCollections = await _collectionRepository.GetManyByOrganizationIdAsync(orgId);
+            orgCollections = await _collectionRepository.GetManySharedCollectionsByOrganizationIdAsync(orgId);
         }
         else
```

Comment:
I believe the `else` block is used to display collections in AC when the user cannot read all collections - so they just see the collections they're assigned to. In this case, do we want the default collection included? I don't think so, but please check with Priya. You may have to filter out the default collection in the `else` block if not.

That said - I'm not sure when this actually happens - can you view the Collections tab but not view all collections?

This is not strictly required for the fix here so you can look at it separately if you want.